### PR TITLE
Request to run 2014-09.

### PR DESCRIPTION
**Pull request by kdraeder**
pre_submit.csh updated with more complete instructions
for proceeding.

### CODE REVIEW

File: pre_submit.csh [hunk 1]
```diff
@@ -201,8 +204,17 @@ sort cam.input_data_list | uniq > cam.input_data_list.sorted
 sort clm.input_data_list | uniq > clm.input_data_list.sorted
 cd -
 
+@ new_month = $data_month + 1
+echo "Now issue a pull request to DART_CASES:"
+echo "   % git status -uno > run_req_${data_year}_${new_month}.csh"
+echo "   Edit run_ to make it `git add` all of the modified files"
+echo "   % csh run_req_YYYY-MM.csh"
```
Comment (timhoar) [inline pre_submit.csh]: If this is now part of the normal process, run_req_YYYY-MM.csh to the git repo.

**Comment by timhoar** [comment]
It seems like you have not added the 'unsorted' [cam,clm]_file_list. 
Now that we routinely produce the sorted list, I'd say we can do a 'git rm' on those unsorted files.

File: pre_submit.csh (hunk 1 above)
Comment (nancycollins) [reply pre_submit.csh]: if all the lines in the output need git add, you could run:

git status -uno | sed -e 's/^/git add /' > run_xxx

and avoid mindless editing.

Comment (kdraeder) [reply pre_submit.csh]: I meant to replace with ${data_year}_${new_month}.
I'll fix that and push again.

**Comment by kdraeder** [comment]
I believe that I `git rm`ed the unsorted _file_list files  in the last PR,
which is why they don't show up in this PR.

**Comment by timhoar** [comment]
I see that you deleted those file in PR#14 ... good. I remember blowing
past them now.

I have changed my mind about preserving run_req_YYYY-MM.csh ... all this
does is help you look at the files you are about to commit.
The commit also has that information - so it is one less file to add.
Personally - I don't know why you wouldn't just call this file 'bob.csh'


Tim Hoar
Data Assimilation Research Section
National Center for Atmospheric Research
thoar@ucar.edu
303.497.1708

File: pre_submit.csh (hunk 1 above)
Comment (timhoar) [reply pre_submit.csh]: run_req_YYYY-MM.csh  is (in my mind) a disposable file, now that I think about it.
No need to add it to the git repo.

Nancy -- visually inspecting the contents of the file is a good idea - and if it takes me a second to do a global search/replace inside the file I need to look at anyway - it's not enough to worry about. I think it's the same amount of keystrokes if you do it with sed on the command line or vi inside the file.

**Review by timhoar — APPROVED** [review]

File: pre_submit.csh [hunk 2]
```diff
@@ -201,8 +202,18 @@ sort cam.input_data_list | uniq > cam.input_data_list.sorted
 sort clm.input_data_list | uniq > clm.input_data_list.sorted
 cd -
 
+@ new_month = $data_month + 1
+echo "Now issue a pull request to DART_CASES:"
+echo "   % git status -uno >! push_prep.csh"
+echo "   % echo 'git add push_prep.csh' >> push_prep.csh"
```
Comment (timhoar) [inline pre_submit.csh]: Nope ... do not add push_prep.csh to the repo.

**Review by timhoar — APPROVED** [review]

**Pull request merged**